Modules/Core/SpatialObjects/include/itkContourSpatialObject.h

```diff
@@ -158,18 +158,18 @@ class ITK_TEMPLATE_EXPORT ContourSpatialObject
   /** Set the interpolation type */
   itkSetEnumMacro(InterpolationMethod, InterpolationMethodEnum)
```

Comment:
Can you add a ';' at the end of the macro to get indenting correct?

Reply:
Awesome catch Hans :eagle:. Done in da7f9f4. `clang-format` is happy now.